fix: Handle Empty State for KeyValueEntry (#11129)
laravel-filament/filament: packages/infolists/docs/03-entries/06-key-value.md

KeyValueEntry::make('meta')
    ->valueLabel('Property value')
```

> ## Customizing the empty value pair's label
> 
> You may customize the message for empty value pairs using the `emptyMessage()` method:

Reviewer: ```suggestion
## Customizing the empty message

You may customize the message that is displayed when the entry is empty using the `emptyMessage()` method:
```
